source/ngf-vk/impl.c

```diff
@@ -4263,88 +4334,62 @@ ngf_error ngf_create_render_target(const ngf_render_target_info* info, ngf_rende
   ngf_error err    = NGF_ERROR_OK;
   VkResult  vk_err = VK_SUCCESS;
 
+  bool     is_multisample       = false;
+  uint32_t ncolor_attachments   = 0u;
+  uint32_t nresolve_attachments = 0u;
+
+  for (uint32_t a = 0u; a < info->attachment_descriptions->ndescs; ++a) {
+    if (info->attachment_descriptions->descs[a].sample_count > NGF_SAMPLE_COUNT_1) {
+      is_multisample = true;
+    }
+
+    if (info->attachment_descriptions->descs[a].type == NGF_ATTACHMENT_COLOR) {
+      if (info->attachment_descriptions->descs[a].is_resolve) {
+        ++nresolve_attachments;
+      } else {
+        ++ncolor_attachments;
+      }
+    }
+  }
+
+  if (is_multisample && nresolve_attachments == 0) {
```

Comment:
it's actually totally legal to have an render target that only has multisampled attachments. we should remove this check.